SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -621,6 +621,17 @@ void USpatialActorChannel::UpdateSpatialPosition()
 
 void USpatialActorChannel::UpdateSpatialRotation()
 {
+	FRotator ActorSpatialRotation = Actor->GetActorRotation();
+
+	// Check that it's rotated sufficiently far to be updated.
+	const float SpatialRotationThreshold = 0.1f;  // 0.1 radian (~5.7 degrees)
+	FRotator RotationDelta = ActorSpatialRotation - LastSpatialRotation;
```

Comment:
const FRotator

Reply:
Replaced with a quaternion that gets normalized in place, so can't be const.